--- autogen.sh	2013-02-23 03:40:55.000000000 +0000
+++ autogen.sh.NEW	2013-02-23 03:43:11.000000000 +0000
@@ -5,9 +5,10 @@
 AM_INSTALLED_VERSION=$($AUTOMAKE --version | sed -e '2,$ d' -e 's/.* \([0-9]*\.[0-9]*\).*/\1/')
 
 if [ "$AM_INSTALLED_VERSION" != "1.10" \
-    -a "$AM_INSTALLED_VERSION" != "1.11" ];then
+    -a "$AM_INSTALLED_VERSION" != "1.12" \
+    -a "$AM_INSTALLED_VERSION" != "1.13" ];then
 	echo
-	echo "You must have automake > 1.10 or 1.11 installed to compile lxpanel."
+	echo "You must have automake >=1.10 installed to compile lxpanel."
 	echo "Install the appropriate package for your distribution,"
 	echo "or get the source tarball at http://ftp.gnu.org/gnu/automake/"
 	exit 1
